Combinatorial_map/doc/Combinatorial_map/Concepts/GenericMapItems.h

```diff
@@ -2,16 +2,15 @@
 \ingroup PkgCombinatorialMapsConcepts
 \cgalConcept
 
-The concept `GenericMapItems` allows to customize a <I>d</I>D generic map by choosing the information associated with darts, and by enabling and disabling some attributes. For that, it defines an inner class template named
-\link GenericMapItems::Dart_wrapper `Dart_wrapper`\endlink, with one template parameter, `Map`, a model of the `GenericMap` concept. This inner class can define the two types `%Dart_info` and `%Attributes`.
+The concept `GenericMapItems` allows to customize a <I>d</I>D generic map by choosing the information associated with darts, by enabling and disabling some attributes, \tred{and by choosing to use indices or handles}. For that, it defines an inner class template named \link GenericMapItems::Dart_wrapper `Dart_wrapper`\endlink, with one template parameter, `Map`, a model of the `GenericMap` concept. This inner class can define the two types `%Dart_info` and `%Attributes`.
 
 \cgalHasModel \link CGAL::Generic_map_min_items `CGAL::Generic_map_min_items`\endlink
 
 \sa `GenericMap`
 
 \cgalHeading{Example}
 
-The following examples show two possible models of the `GenericMapItems` concept: the first one for a generic map without dart information, nor enabled attributes, the second one for a generic map with a `double` associated with each dart, and edge attributes enabled, and associated with a \link CGAL::Cell_attribute `Cell_attribute`\endlink containing an `int`.
+The following examples show two possible models of the `GenericMapItems` concept: the first one for a generic map without dart information, nor enabled attributes, the second one for a generic map \tred{that uses std::uint16_t as indices,} with a `double` associated with each dart, and edge attributes enabled, and associated with a \link CGAL::Cell_attribute `Cell_attribute`\endlink containing an `int`.
```

Comment:
Seen the warning in the overnight build (see: https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/ e.g. https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-5.6-Ic-29/index.html and https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-5.6-Ic-29/logs_master/Combinatorial_map.log, note that the line numbers are not 100% accurate) it looks like that the command `\tred` is not defined in the `ALIASES` or maybe something else is meant. When a command for the `ALIASES` is meant it is probably better to rename it to something like `cgalTred`.